okta/framework/ApiClient.py

```diff
@@ -9,8 +9,16 @@
 class ApiClient(object):
 
     def __init__(self, *args, **kwargs):
-        self.base_url = kwargs['base_url'] or args[0]
-        self.api_token = kwargs['api_token'] or args[1]
+        if 'pathname' not in kwargs:
+            raise ValueError('Invalid url')
+
+        if 'base_url' in kwargs and 'api_token' in kwargs:
```

Comment:
This isn't equivalent to what we had before, so may (unlikely, but may) break consumers. What if they were passing a `base_url` in the `*args`, but an `api_token` in `*kwargs`? We should split this into two `if`/`else` clauses: a `base_url` check and an `api_token` check.